src/AddImageChecks.cpp

```diff
@@ -251,7 +251,7 @@ Stmt add_image_checks(Stmt s,
             Expr actual_extent = Variable::make(Int(32), actual_extent_name, image, param, rdom);
             Expr actual_stride = Variable::make(Int(32), actual_stride_name, image, param, rdom);
             if (!touched[j].is_bounded()) {
-                user_error << "Buffer " << name
+                user_error << "BufferPtr " << name
```

Comment:
I think it should remain `Buffer` in the error message. It's the buffer being accessed, not the pointer to the buffer (I'm guessing this was a find/replace artifact...).